<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTextView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_FileOpen"/>
    <addaction name="action_FileOpenNext"/>
    <addaction name="action_FileOpenPrevious"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_TextCopy"/>
   </widget>
   <widget class="QMenu" name="menu_Search">
    <property name="title">
     <string>&amp;Search</string>
    </property>
    <addaction name="action_TextSearch"/>
    <addaction name="action_TextSearchNext"/>
    <addaction name="action_TextSearchPrevious"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_AboutQt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Search"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_FileOpen"/>
   <addaction name="action_FileOpenPrevious"/>
   <addaction name="action_FileOpenNext"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_FileOpen">
   <property name="icon">
    <iconset resource="qtextview.qrc">
     <normaloff>:/img/icons/oxygen/16/document-open.png</normaloff>:/img/icons/oxygen/16/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="qtextview.qrc">
     <normaloff>:/img/icons/oxygen/16/application-exit.png</normaloff>:/img/icons/oxygen/16/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_TextCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qtextview.qrc">
     <normaloff>:/img/icons/oxygen/16/edit-copy.png</normaloff>:/img/icons/oxygen/16/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_TextSearch">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_TextSearchNext">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Search next</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_TextSearchPrevious">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Search previous</string>
   </property>
   <property name="shortcut">
    <string>Shift+F3</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="qtextview.qrc">
     <normaloff>:/img/icons/oxygen/16/help-about.png</normaloff>:/img/icons/oxygen/16/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_AboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="action_FileOpenNext">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qtextview.qrc">
     <normaloff>:/img/icons/oxygen/16/go-next.png</normaloff>:/img/icons/oxygen/16/go-next.png</iconset>
   </property>
   <property name="text">
    <string>Open next</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="action_FileOpenPrevious">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qtextview.qrc">
     <normaloff>:/img/icons/oxygen/16/go-previous.png</normaloff>:/img/icons/oxygen/16/go-previous.png</iconset>
   </property>
   <property name="text">
    <string>Open previous</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtextview.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
